bot.py

```diff
@@ -35,5 +35,7 @@ async def on_message(msg):
 
   }
   return switchcase.get(cmd, None)
-
-client.run(token)
+if __name__ == "__main__":
```

Comment:
This is not needed as it is only one line that we need to run. this is just extra lines that we dont need.

Reply:
That's just fine. It is a practice. Not something that is going to change functionality